src/main/resources/develocity/gradle/develocity-init-script.gradle

```diff
@@ -142,12 +173,12 @@ if (GradleVersion.current() < GradleVersion.version('6.0')) {
                         buildScanExtension.publishAlways()
                     }
                     // uploadInBackground not available for build-scan-plugin 1.16
-                    if (buildScanExtension.metaClass.respondsTo(buildScanExtension, 'setUploadInBackground', Boolean)) buildScanExtension.uploadInBackground = false
```

Comment:
For CI injection use case, we want `uploadInBackground` to always be off. 
Is there a use case to have this being configurable?

Reply:
This value is currently configurable in the init-script for the GitHub Actions, Jenkins and Teamcity integrations. It's not clear to me why this was added or whether there's a use case for it being configurable.

This change is simply to make the Bamboo plugin init-script consistent with other CI integrations, in order to make a single reference implementation possible.

If we decide to remove this, we should do it consistently for all CI integrations.

Reply:
Ok, understood, let's have it in sync then.